Shared/Components/AlphaPicker/AlphaPickerButton.swift

```diff
@@ -0,0 +1,105 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2023 Jellyfin & Jellyfin Contributors
+//
+
+// Joe Kribs: joseph@kribs.net 02/06/2023
+// Formats the Letters in the LetteredScrollbar
+
+import Defaults
+import SwiftUI
+
+extension AlphaPickerView {
+
+    struct AlphaPickerButton: View {
+
+        @Default(.accentColor)
+        private var accentColor
+
+        private let letter: Character
+        private let letterFont: Font
```

Comment:
The font, frame, padding, and highlight properties aren't how UI values are passed in SwiftUI. Instead, use the `.font()` and `.padding()` modifiers which are passed into the button's environment. I also don't see value in the manual padding and highlight padding.

For font sizes, please use the defined font sizes as defined by the HIG, only in a few places are the fonts manually defined but should be moved over (a place in LiveTV).

- https://developer.apple.com/design/human-interface-guidelines/typography#Specifications

Reply:
A lot of this came from an older version of this I was working on with the goal of reusing this class for both iOS/iPadOS & tvOS. Looking back at it, this would likely create a subpar experience on 2 platforms. I'll clear these out!